commands/services/tail.ts

```diff
@@ -59,7 +59,7 @@ export const servicesTailCommand =
         let writer: WritableStreamDefaultWriter<string | Uint8Array>;
   
         try {
-          conn = await stream.connection;
+          conn = await stream.opened;
```

Comment:
@shawn-render I removed the check for stream.connection since it was failing the compile stage. The previous check was unnecessary since the compile stage will bundle the appropriate version of Deno anyway